code/modules/mob/living/brain/brain_item.dm

```diff
@@ -37,7 +37,7 @@
 		if(C.has_brain_worms())
 			var/mob/living/simple_animal/borer/B = C.has_brain_worms()
 			B.leave_victim() //Should remove borer if the brain is removed - RR
-		transfer_identity(C)
+	transfer_identity(C)
```

Comment:
Yeah it should take the fake one, surgery wise, as the real changeling's brain is "in the chest" (it's not but it's certainly not the head) so being able to do surgery on the head and getting the real one is... weird and wrong.